File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/factory/transportmethod/Helicopter.java
package com.bigboxcode.designpattern.factory.transportmethod;

public class Helicopter implements Transport {
        @Override
        public void start() {
            System.out.println("Helicopter started");
        }

        @Override
        public void stop() {
            System.out.println("Helicopter Stopped");
        }

        @Override
        public void repair() {
            System.out.println("Helicopter Repair");
        }
}

File: design-patterns/php/factory/transportmethod/Helicopter.php
<?php
// Helicopter.php

namespace BigBoxCode\DesignPattern\Factory\TransportMethod;

class Helicopter implements Transport {
    public function start() {
        echo "Helicopter started\n";
    }

    public function stop() {
        echo "Helicopter Stopped\n";
    }

    public function repair() {
        echo "Helicopter Repair\n";
    }
}